deny.toml

```diff
@@ -51,7 +51,7 @@ ignore = false
 registries = []
 
 [bans]
-multiple-versions = "warn"
+multiple-versions = "allow"
```

Comment:
We inherently depend on multiple versions (because of multivm), so it cannot realistically be fixed without a humongous effort.